### ISSUE TITLE
Disabling "Watch for changes" and/or rescan interval not respected

### ISSUE THREAD

**Issue by tristanbrown** [syncthing/syncthing#6189]
For some of my folders, I've disabled "Watch for changes" and set the rescan interval to 7200. I have a "Send Only" machine and a "Receive Only" machine. 

For some reason, these rescan preferences are not respected here. It behaves as though watching is still enabled and rescans every 2 seconds, on every file change. I have tried creating several different folders on this machine, with the same results. 

This is only an issue for one of my sender machines. I have another sender machine configured exactly the same way, and the rescans happen on the correct interval on that one. At this point, I cannot imagine what the difference could be. 

### Version Information

Syncthing Version: v1.3.1
SyncTrayzor Version: v1.1.23
OS Version: Windows 10

**Comment by uok** [comment]
You can take a look at the config.xml and compare the settings for the folders directly.
Maybe a setting has not been saved correctly.

**Comment by tristanbrown** [comment]
Yes, the config.xml entries are identical.

**Comment by calmh** [comment]
Syncthing restarted? (I could imagine a bug where the filesystem watcher wasn't correctly shut down on config change, if it was enabled before.)

**Comment by tristanbrown** [comment]
Yes, Syncthing restarted on both the sending and receiving machines.

**Comment by tristanbrown** [comment]
I've found the source of the problem. Though the config.xml files located at %APPDATA%\Local\Syncthing\ were identical between both machines (with fsWatcherEnabled="False" for the folder in question), I found the discrepancy in the config.xml file located at %APPDATA%\Roaming\SyncTrayzor\. Here, the <IsWatched> attribute was set to "True" for every folder on the offending machine, and "False" on the machine that is working as intended. When I switch this flag to "False" for all of the folders on the offending machine, it works as intended. 

So it seems the issue is that for newer installs, SyncTrayzor folder-watching is on by default, until it is manually disabled from the File>Settings menu. Disabling "Watch for Changes" in the "Edit Folder" dialogue only disables Syncthing folder-watching, not SyncTrayzor folder-watching. This behavior is very unintuitive, and not indicated in any way by the GUI.
